scripts/docs_gates/smoke.py

```
@@ -0,0 +1,104 @@
+    return SEARCH_MOUNT_MARKER in html
+
+
+def run(host: str, slug: str, expect_sha: str, access_id: str, access_secret: str,
+        pdf: str | None, critical: list[str]) -> int:
+    failures = 0
+    for probe in probe_plan(slug, pdf, critical):
+        req = urllib.request.Request(f"https://{host}{probe.path}", method="GET")
+        req.add_header("CF-Access-Client-Id", access_id)
+        req.add_header("CF-Access-Client-Secret", access_secret)
+        try:
+            with urllib.request.urlopen(req, timeout=30) as resp:
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)